src/main/scala/dpla/ingestion3/mappers/providers/CdlExtractor.scala

```diff
@@ -0,0 +1,84 @@
+package dpla.ingestion3.mappers.providers
+
+import java.net.URI
+
+import dpla.ingestion3.mappers.json.JsonExtractionUtils
+import dpla.ingestion3.model.{DplaMapData, DplaSourceResource, EdmWebResource, OreAggregation, _}
+import org.json4s._
+import org.json4s.jackson.JsonMethods._
+
+class CdlExtractor extends Extractor with JsonExtractionUtils {
+
+  def build(rawData: String): DplaMapData = {
+
+    implicit val json: JValue = parse(rawData)
```

Comment:
So I don't have to pass it as an argument explicitly. Makes the following code cleaner looking.